### DISCUSSION TITLE
About a Button for removal 'Recommended based on people you follow'

### DISCUSSION THREAD

**Discussion by yangchoi** [community/community#29533] [Other Feature Feedback, Questions, & Ideas]
Hi, I have a discussion about 'for you'.
It's about 'Recommended based on people you follow' Box. 
I think there are same account from spring until I really follow them. 

I would like to be recommended other person for following, 
and If I don't want this person to follow I hope to remove the box of this person. 

if there is a button x for removing, It would be really helpful.
![image](https://user-images.githubusercontent.com/48708746/184654685-ad417c85-b165-4eb0-8ee8-d07808f253e5.png)

thank you for reading : )

**Comment by thebiltheory** [comment]
@github-staff I think this one is quite important for our mental health 🤣 
I've been seeing the same persons(which I don't necessarly hold in my heart) over and over again every time I open my page.
Some options to discart the suggestion would really help.

**Reply by yangchoi** [reply]
I have same person like you.. so I totally agree 🤣
